helm-charts/templates/data-plane/gateway-components/gateway-runtime/gateway-runtime-deployment.yaml

```diff
@@ -85,6 +87,20 @@ spec:
             {{- else }}
               subPath: tls.crt
             {{- end }}
+            - name: enforcer-keystore-secret-volume
+              mountPath: /home/wso2/security/truststore/apk.crt
+            {{- if and .Values.wso2.apk.dp.gatewayRuntime.deployment.enforcer.configs .Values.wso2.apk.dp.gatewayRuntime.deployment.enforcer.configs.tls }}
+              subPath: {{ .Values.wso2.apk.dp.gatewayRuntime.deployment.enforcer.configs.tls.caCertFileName | default "ca.crt" }}
+            {{- else }}
+              subPath: ca.crt
+            {{- end }}
+            - name: enforcer-keystore-secret-volume
+              mountPath: /home/wso2/security/truststore/enforcer.crt
```

Comment:
why do we add the enforcer public cert to it's own truststore?

Reply:
this is needed for mtls in probe